slurp.py

```diff
@@ -942,18 +959,21 @@ def matches( rule, kwargs={} ):
                 base2 = base1.split('_')[-2:]
                 input_datasets[ '_'.join(base2) ] = 1
 
-    
+   
     if len(lfn_lists)==0: return [], None, []  # Early exit if nothing to be done
     
+
     # Build dictionary of DSTs existing in the datasets table of the file catalog.  For every DST that is in this list,
     # we know that we do not have to produce it if it appears w/in the outputs list.
     dsttype="%s_%s_%s"%(name,build,tag)  # dsttype aka name above
```

Comment:
TODO: May want to use 'name_' in the dsttype... I believe we want streamname substituted here.